bin/install_deps.pl

```diff
@@ -451,10 +451,12 @@ sub name_overrides {
 # couple rules. When that doesn't work, add entries here for FreeBSD (port),
 # MacPorts ($dport), yum, and apt.
     my @modules = (
-        { module=>'LWP::UserAgent', info => { cat=>'www', port=>'p5-libwww', dport=>'p5-libwww-perl', yum=>'perl-libwww-perl' }, },
-        { module=>'Mail::Send'    , info => { port => 'Mail::Tools', }  },
-        { module=>'Date::Parse'   , info => { port => 'TimeDate',    }  },
-        { module=>'LWP'           , info => { port => 'p5-libwww',   }  },
+        { module=>'LWP::UserAgent'    , info => { cat=>'www', port=>'p5-libwww', dport=>'p5-libwww-perl', yum=>'perl-libwww-perl' }, },
+        { module=>'Mail::Send'        , info => { port => 'Mail::Tools', }  },
+        { module=>'Date::Parse'       , info => { port => 'TimeDate',    }  },
+        { module=>'LWP'               , info => { port => 'p5-libwww',   }  },
```

Comment:
@msimerson there's only whitespace changes in those four lines, i.e. lines 454 to 457.